python/tidy/tidy.py

```diff
@@ -684,40 +662,12 @@ def no_filter(match, line) -> bool:
             yield (idx + 1, "found an empty line following a {")
         prev_open_brace = line.endswith("{")
 
-        # check alphabetical order of feature attributes in lib.rs files
-        if is_lib_rs_file:
-            match = re.search(r"#!\[feature\((.*)\)\]", line)
-
-            if match:
-                features = list(map(lambda w: w.strip(), match.group(1).split(",")))
-                sorted_features = sorted(features)
-                if sorted_features != features and check_alphabetical_order:
-                    yield (
-                        idx + 1,
-                        decl_message.format("feature attribute")
-                        + decl_expected.format(tuple(sorted_features))
-                        + decl_found.format(tuple(features)),
-                    )
-
-                if prev_feature_name > sorted_features[0] and check_alphabetical_order:
-                    yield (
-                        idx + 1,
-                        decl_message.format("feature attribute")
-                        + decl_expected.format(prev_feature_name + " after " + sorted_features[0])
-                        + decl_found.format(prev_feature_name + " before " + sorted_features[0]),
-                    )
-
-                prev_feature_name = sorted_features[0]
-            else:
-                # not a feature attribute line, so empty previous name
-                prev_feature_name = ""
-
         if is_panic_not_allowed_rs_file:
             match = re.search(r"unwrap\(|panic!\(", line)
             if match:
                 yield (idx + 1, panic_message)
 
-        # modules must be in the same line and alphabetically sorted
+        # modules must be in the same line
```

Comment:
Does `rustfmt` make this check unnecessary as well? Perhaps you could remove this in a followup if so.